lib/dockerize_rails/docker_commands.rb

```diff
@@ -0,0 +1,35 @@
+module DockerizeRails
+  module DockerCommands
+    require 'docker'
+
+    def self.version
+      Docker.version
+    end
+
+    def self.pull
+      status = 0
+      status += DockerPull.pull_ruby
+      status += DockerPull.pull_mysql
+      status += DockerPull.pull_postgres
+      status
+    rescue Docker::Error::NotFoundError => e
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 'e' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)

<!-- Comment by [SideCI](https://sideci.com) -->